sentry_sdk/tracing.py

```diff
@@ -1002,10 +999,7 @@ def my_function():
         async def my_async_function():
             ...
     """
-    if PY2:
-        from sentry_sdk.tracing_utils_py2 import start_child_span_decorator
-    else:
-        from sentry_sdk.tracing_utils_py3 import start_child_span_decorator
+    from sentry_sdk.tracing_utils_py3 import start_child_span_decorator
```

Comment:
We should probably remove the `_py3` suffix from this files (and other files that have a suffix like this). Although, it might be better to make this change in a separate PR. 

Do you already have an issue/PR open to remove these suffixes?

Reply:
That's coming in a separate PR (not open yet). In general, if a change is part of the [monster PR](https://github.com/getsentry/sentry-python/pull/2581), there will be a PR for it at some point -- trying to make the individual PRs as easy to review as possible.

Reply:
Got it, thanks for sharing the link to the monster PR